etc/release.sh

```diff
@@ -9,8 +9,9 @@ version=${1}
 # Ensure version is non-empty
 [ ! -z "${version}" ] || { echo "ERROR: Missing version string"; exit 1; }
 
-# verify that the examples build
-./etc/build-examples.sh
+# verify that the examples build with the current branch we're releasing on
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
can we use `git branch --show-current` here instead? or is this version required for certain cases?

Reply:
that should work, the stack overflow post I got this from seems to predate the existence of `--show-current` 🙂 updated!